### ISSUE TITLE
False License

### ISSUE THREAD

**Issue by chaos234** [Modbder/EssentialCraft3#50]
Hey,

EE3 uses a false license and so it is under GNU License if no other, correct license is provided.

Q: Why?
A: CC Licenses are only for works like images or music. You should think about to change the license to MIT or let it be GNU but with the point that you must be mentioned as original author.

I hope that this will help you :).

**Comment by thePalindrome** [comment]
Not a lawyer, but I'm fairly certain that's not how it works.

CC is a valid license, albeit without wording that applies well to software. This won't be too much of an issue, except for the fact that software can do things art can't (for example, the liability clause is missing from the CC license, since art can't crash a computer by having a mis-colored pixel)

If you want a general license, MIT works great. A fairly popular license that preserves ownership would be the BSD family of licenses. To make sure that EC2 will always be open source, go for a GPL license.

Alas, in order to change the license you **must** have every single person who contributed code accept the new license. Otherwise it's a weird legal issue.

tl;dr The CC license remains in full effect despite not being software-specific, but it's a legal mess.
